### PR TITLE
FR Translation

### PR DESCRIPTION
**Pull request by MajorFox-MJ**
# SCP:SL Translation PR Template

## Language
(Replace this text with language name)

## Type of change
- [ ] Added new language translation
- [X] Changed a currently existing translation

## Have you tested the changes you've made?
- [X] Yes
- [ ] No

# Checklist:
- [X] I have made sure to not commit changes to other stuff than my own or the default English one
- [X] I have run `EncodingNormalizer.exe` on my changed files before committing 
   -  (Exception to this is when edits have __only__ been made in-browser)
- [X] I have made sure that I'm not submitting a duplicate translation 
- [X] I have not included any advertisements/pointers to non-game stuff other than the author in the folder name

### CODE REVIEW

**Comment by Dat-Joseph** [comment]
be sure to delete the translation files in your pull request. We dont use them anymore :)

File: Translations/fr/Class_Nicknames.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+-
+-
+Le Vieil Homme
+-
+Le Docteur
+-
+L'IA
+-
+L'Homme Timide
+Zombie
```
Comment (Dat-Joseph) [inline Translations/fr/Class_Nicknames.txt]: I dont think we should let it be "Zombie" edit it back to "The cured" if possible. If you refer 049-2 as Zombie in any other files, please edit them too.

Comment (MajorFox-MJ) [reply Translations/fr/Class_Nicknames.txt]: This is just the nickname, I didn't change anything else

Comment (Dat-Joseph) [reply Translations/fr/Class_Nicknames.txt]: Then it probably was the nickname too before you corrected the translation. I still would prefer a different nickname other than "zombie"

File: Translations/fr/Death_Causes.txt [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+Électrocuté.
+Écrasé par une structure lourde.
+Il y a beaucoup de blessures par balles sur le corps, c'est probablement du <color=#ffff00>[ammotype]</color> qui a été utilisée.
+<color=#ff0000>Nuque brisée à la base du crâne.</color>
+<color=#ff0000>N'a pas survécu à sa rencontre avec SCP-106.</color>
+<color=#ff0000>Soigné de la peste.</color>
+La cause de la mort est inconnue.
+Il s'agit du corps d'un [class] nommé <b>[user]</b> !\n\nCause de la mort : [cause]
+<color=#ff0000>Mis en pièces par SCP-096.</color>
+<color=#ff0000>Morsure puissante à la base du crâne.</color>
```
Comment (Dat-Joseph) [inline Translations/fr/Death_Causes.txt]: I do not see any color codes in the english translation, remove them too :)

File: Translations/fr/Legacy_Interfaces.txt [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+ogive Alpha
+Activée
+Désactivée
+La prochaine partie débutera dans [time] secondes.
+CAPACITÉ SPÉCIALE
+CRÉER UN PORTAIL
+UTILISER LE PORTAIL
+VOUS ÊTES MORT
+<size=30><color=#ff0000>Vous avez été tué par :</color></size>
+<size=30><color=#ffff00>en tant que :</color></size>
```
Comment (Dat-Joseph) [inline Translations/fr/Legacy_Interfaces.txt]: Same goes for these. Color codes need to be removed.

**Comment by MichalPetryka** [comment]
Don't use upload files

File: Translations/fr/Death_Causes.txt (hunk 2 above)
Comment (MajorFox-MJ) [reply Translations/fr/Death_Causes.txt]: Aye

File: Translations/fr/Legacy_Interfaces.txt (hunk 3 above)
Comment (MajorFox-MJ) [reply Translations/fr/Legacy_Interfaces.txt]: Thanks :3

**Pull request closed without merging**